### DISCUSSION TITLE
How to participate in the development of applehv?

### DISCUSSION THREAD

**Discussion by BlackHole1** [containers/podman#20719] [General]
I would like to contribute to the development of podman in supporting applehv, but I have encountered some issues. I am unable to fetch the base image for development. Please refer to the image below.

![37-20 17 37@2x](https://github.com/containers/podman/assets/8198408/5c78c5b8-0edf-4793-ae7f-b5c1b87c9e8f)

Can you provide the online address of the image? So that I can specify it using `podman machine init --image-path`.

/cc @baude

**Comment by Luap99** [comment]
There is https://fedorapeople.org/groups/podman/testing/applehv/, not sure if it is up to date though. @baude would know

**Reply by BlackHole1** [reply]
Thanks!
